Fix for long long time_t on LP64 archs. Poco's type system assumes that
64-bit quantities are type long on LP64 archs and that long long
only exists on 32-bit archs.

--- Foundation/src/PatternFormatter.cpp.orig	Sat Aug 17 16:38:20 2013
+++ Foundation/src/PatternFormatter.cpp	Sat Aug 17 16:38:55 2013
@@ -123,7 +123,7 @@ void PatternFormatter::format(const Message& msg, std:
 				case 'F': NumberFormatter::append0(text, dateTime.millisecond()*1000 + dateTime.microsecond(), 6); break;
 				case 'z': text.append(DateTimeFormatter::tzdISO(_localTime ? Timezone::tzd() : DateTimeFormatter::UTC)); break;
 				case 'Z': text.append(DateTimeFormatter::tzdRFC(_localTime ? Timezone::tzd() : DateTimeFormatter::UTC)); break;
-				case 'E': NumberFormatter::append(text, msg.getTime().epochTime()); break;
+				case 'E': NumberFormatter::append(text, (Int64)msg.getTime().epochTime()); break;
 				case '[':
 				{
 					++it;
